pyproject.toml

```diff
@@ -1,63 +1,95 @@
 [build-system]
-requires = [
-    "setuptools>=61.0.0",
-    "setuptools-scm>=8.0",
-]
+requires = ["setuptools>=61.0.0", "setuptools-scm>=8.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.9",
+    "License :: OSI Approved :: Apache Software License",
+]
 name = "nomad-parser-magres"
+description = "NOMAD parser plugin for the magres file format."
 dynamic = ["version"]
-description = "A NOMAD plugin for parsing magres files."
 readme = "README.md"
+requires-python = ">=3.9"
 authors = [
-    { name = "Jose M. Pizarro", email = "jose.pizarro@physik.hu-berlin.de" }
+    { name = "The CCP-NC Authors" },
 ]
-license = { text = "Apache-2.0" }
-requires-python = ">=3.9"
-classifiers = [
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.9",
-    "License :: OSI Approved :: Apache Software License",
+maintainers = [
+    { name = "The CCP-NC Authors" },
```

Comment:
So it seems that this does not for the PyPI page. I thought it would show these and the authors as a list, but we can keep it as it is now due to its little impact